website/templatetags/customfilters.py

```diff
@@ -1,5 +1,6 @@
 from django import template
 import re
+import markdown
```

Comment:
```re``` is a system package. Should be first. Then, blank line. Then, import django, and the markdown.